src/backend/api/Fusion.Resources.Api/Notifications/InternalRequestNotification/RequestInitializedHandler.cs

```diff
@@ -20,65 +20,76 @@ public partial class InternalRequestNotification
+                                    .AddFact("Position", request.Position!.Name)
+                                    .AddFact("Period", $"{request.Instance?.AppliesFrom:dd.MM.yyyy} - {request.Instance?.AppliesTo:dd.MM.yyyy}") // Until we have resolved date formatting issue related to timezone.
+                                    .AddFact("Workload", $"{request.Instance?.Workload}")
                                 )
-                                .TryAddOpenPortalUrlAction("Open request", $"{request.PortalUrl}")
-                                ;
-                        });
+                                .AddTextBlock($"Created by: {request.AllocationRequest.CreatedBy.Name}")
+                                .TryAddOpenPortalUrlAction("Open position in org admin", $"{request.OrgAdminUrl}");
+
                     }
                     else // Person assigned, try to add profile card
                     {
-                        await notificationClient.CreateNotificationForUserAsync(recipient, arguments, builder =>
-                        {
-                            builder
+                        notificationBuilder
+                                .AddTitle("A request workflow has been initialized")
                                 .TryAddProfileCard(request.Instance.AssignedPerson.AzureUniqueId)
-                                .AddDescription(
-                                    $"{request.Instance.AssignedPerson.Name} ({request.Instance.AssignedPerson.Mail}) was proposed for position {request.Position.Name}.")
+                                .AddDescription($"{request.Instance.AssignedPerson.Name} ({request.Instance.AssignedPerson.Mail}) was proposed for position {request.Position.Name}.")
+
                                 .AddFacts(facts => facts
-                                    .AddFact("Project", request.Position.Project.Name)
-                                    .AddFact("Request created by", request.AllocationRequest.CreatedBy.Name)
+                                    .AddFactIf("Project", request.Position.Project.Name, request.Position?.Project != null)
+                                    .AddFact("Position", request.Position!.Name)
+                                    .AddFact("Period", $"{request.Instance?.AppliesFrom:dd.MM.yyyy} - {request.Instance?.AppliesTo:dd.MM.yyyy}") // Until we have resolved date formatting issue related to timezone.
+                                    .AddFact("Workload", $"{request.Instance?.Workload}")
                                 )
-                                .TryAddOpenPortalUrlAction("Open request", $"{request.PortalUrl}")
+                                .AddTextBlock($"Created by: {request.AllocationRequest.CreatedBy.Name}")
+                                .TryAddOpenPortalUrlAction("Open position in org admin", $"{request.OrgAdminUrl}")
                                 ;
-                        });
+
```

Comment:
Should these templates be different except for the profile card? if we could just add the profile card to the builder when assigned person is not  null. But YMMV